gpytorch/kernels/_utils.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import torch
+from gpytorch.priors import SmoothedBoxPrior
+
+logger = logging.getLogger()
+
+
+def _bounds_to_prior(
```

Comment:
Can we delete this file since this function appears to be duplicated in `priors._compatability`?

Reply:
good catch, will kill the whole _utils.py file.